docs/Options.md

```diff
@@ -25,21 +25,78 @@ All options need to be defined as environment variables in the environment that
 
 ## List of options
 
-### `LD_SUPERUSER_NAME`
+## Database Options
 
-Values: `String` | Default = None
+### `LD_DB_ENGINE`
 
-When set, creates an initial superuser with the specified username when starting the container.
-Does nothing if the user already exists.
+Values: `postgres` or `sqlite` | Default = `sqlite`
 
-See [`LD_SUPERUSER_PASSWORD`](#ld_superuser_password) on how to configure the respective password.
+Database engine used by linkding to store data.
+Currently, linkding supports SQLite and PostgreSQL.
+By default, linkding uses SQLite, for which you don't need to configure anything.
+All the other database variables below are only required for configured PostgresSQL.
 
-### `LD_SUPERUSER_PASSWORD`
+### `LD_DB_DATABASE`
+
+Values: `String` | Default =  `linkding`
+
+The name of the database. 
+
+### `LD_DB_USER`
+
+Values: `String` | Default =  `linkding`
+
+The name of the user to connect to the database server.
+
+### `LD_DB_PASSWORD`
+
+Values: `String` | Default =  None
+
+The password of the user to connect to the database server.
+The password must be configured when using a database other than SQLite, there is no default value.
+
+### `LD_DB_HOST`
+
+Values: `String` | Default =  `localhost`
+
+The hostname or IP of the database server.
+
+### `LD_DB_PORT`
+
+Values: `Integer` | Default =  None
+
+The port of the database server.
+Should use the default port if left empty, for example `5432` for PostgresSQL.
+
+### `LD_DB_OPTIONS`
+
+Values: `String` | Default = `{}`
+
+A json string with additional options for the database. Passed directly to OPTIONS.
+
+### `LD_FAVICON_PROVIDER`
```

Comment:
Let's move this out of database options. I'd suggest to have a "Other options" or some other catch all category.